ddtrace/contrib/django/utils.py

```diff
@@ -303,6 +307,22 @@ def _after_request_tags(pin, span, request, response):
             raw_uri = url
             if raw_uri and request.META.get("QUERY_STRING"):
                 raw_uri += "?" + request.META["QUERY_STRING"]
+
+            content_type = request.content_type
+            rest_framework = hasattr(request, 'data')
+            print("XXX rest framework: %s" % rest_framework)
+
+            req_body = None
+            if request.method in _BODY_METHODS:
+                if content_type == "application/x-www-form-urlencoded":
+                    req_body = request.data.dict() if rest_framework else request.POST.dict()
+                elif content_type == "application/json":
+                    req_body = json.loads(request.body.decode("UTF-8")) if rest_framework else \
+                        json.loads(request.body.decode("UTF-8"))
```

Comment:
Is there a place where Django parses it too? We would avoid double parsing

Reply:
Not that I know about but I'll double check just in case they added something about this on later versions.

Reply:
I was checking Sqreen's agent code, seems we did this parsing too